### PR TITLE
Update product readme

### PR DESCRIPTION
**Pull request by tutn-axonivy**

### CODE REVIEW

File: adobe-acrobat-sign-connector-product/README.md [hunk 1]
```diff
@@ -7,25 +7,6 @@ The Adobe Acrobat Sign Connector is a project that simplifies the authentication
 
 An Adobe Sign account needs to be created to setup and use the connector.
```
Comment (ivy-rew) [inline adobe-acrobat-sign-connector-product/README.md:8]: I think this sentence should be the first one after ## Adobe Sign account creation

File: adobe-acrobat-sign-connector-product/README.md [hunk 2]
```diff
@@ -52,6 +33,26 @@ The demo project can be used to test the authentication and signing and the demo
...
+ 	 ![fill-birth-date](images/createAccountBirthDate.png)
+
+ 	 d. You will receive a verification code. Insert the code. Will automatically continue
+ 	 ![verification-code](images/createAccountVerificationCode.png)
+
+ 	 e. Developer account is created
+ 	 ![account-finished](images/createAccountFinished.png)
+
+
 Adobe Sign provides 2 options for authentication. (See Setup section)
```
Comment (ivy-rew) [inline adobe-acrobat-sign-connector-product/README.md:56]: this part ... until #Admin Setup Page ... is also somewhat asorted and doesn't help me. I think it should be under an own header? :thinking:  ... but I can't find a good name.

Comment (ivy-rew) [reply adobe-acrobat-sign-connector-product/README.md:57]: also ok for me to leave it as is, maybe @ivy-sgi  or one of the original developers should re-work the full setup description :shrug:

**Review by ivy-rew — APPROVED** [review]

**Comment by ivy-rew** [comment]
thanks a lot @tutn-axonivy

**Comment by tutn-axonivy** [comment]
Hi @ivy-rew, do we need to run the Release script?

**Comment by ivy-rew** [comment]
> Hi @ivy-rew, do we need to run the Release script?

yes, changed product descriptions always require a new release.

**Pull request merged**